src/handlers/notification/callbacks.js

```diff
@@ -52,7 +51,7 @@ const Enum = require('../../lib/enum')
 const sendCallback = async (url, method, headers, message, cid, sourceFsp, destinationFsp) => {
   // validate incoming request parameters are not null or undefined
   if (!url || !method || !headers || !message || !cid || !sourceFsp || !destinationFsp) {
-    throw new Error(Enum.errorMessages.MISSINGFUNCTIONPARAMETERS)
+    throw new Error('Missing parameters for function')
```

Comment:
Please switch to sendRequest from shared library

Reply:
So, remove and replace:

const sendCallback = async (url, method, headers, message, cid, sourceFsp, destinationFsp) => {
  // validate incoming request parameters are not null or undefined
  if (!url || !method || !headers || !message || !cid || !sourceFsp || !destinationFsp) {
    throw new Error('Missing parameters for function')
  }

with sendRequest from shared lib?

Reply:
Addressed, file deleted in order to use the sendRequest from shared lib.